.github/workflows/update_images.yml

```diff
@@ -89,9 +89,11 @@ jobs:
           helm repo add minio https://charts.min.io
           helm dependency build ./charts/tensorleap
           rm ./charts/tensorleap/Chart.lock
-          helm template ./charts/tensorleap \
+          helm dependency build ./charts/tensorleap-infra
+          rm ./charts/tensorleap-infra/Chart.lock
```

Comment:
I'm not sure we want to remove the lock file

Reply:
I will keep it until we will check if it have any influence on chart releases